test/test_grid.py

```diff
@@ -837,27 +840,80 @@ def test_node_face_connectivity_from_files(self):
             grid_xr = xr.open_dataset(grid_path)
             grid_ux = ux.Grid(grid_xr)
 
-            # use the dictionary method to build the node_face_connectivity
-            node_face_connectivity = {}
-            nNodes_per_face = grid_ux.nNodes_per_face.values
-            face_nodes = grid_ux._ds["Mesh2_face_nodes"].values
-            for face_idx, max_nodes in enumerate(nNodes_per_face):
-                cur_face_nodes = face_nodes[face_idx, 0:max_nodes]
-                for j in cur_face_nodes:
-                    if j not in node_face_connectivity:
-                        node_face_connectivity[j] = []
-                    node_face_connectivity[j].append(face_idx)
-
-            # compare the two methods
-            for i in range(grid_ux.nMesh2_node):
-                face_index_from_sparse_matrix = grid_ux.Mesh2_node_faces.values[
-                    i]
-                valid_face_index_from_sparse_matrix = face_index_from_sparse_matrix[
-                    face_index_from_sparse_matrix !=
-                    grid_ux.Mesh2_node_faces.attrs["_FillValue"]]
-                valid_face_index_from_sparse_matrix.sort()
-                face_index_from_dict = node_face_connectivity[i]
-                face_index_from_dict.sort()
-                self.assertTrue(
-                    np.array_equal(valid_face_index_from_sparse_matrix,
-                                   face_index_from_dict))
+        # use the dictionary method to build the node_face_connectivity
+        node_face_connectivity = {}
+        nNodes_per_face = grid_ux.nNodes_per_face.values
+        face_nodes = grid_ux._ds["Mesh2_face_nodes"].values
+        for face_idx, max_nodes in enumerate(nNodes_per_face):
+            cur_face_nodes = face_nodes[face_idx, 0:max_nodes]
+            for j in cur_face_nodes:
+                if j not in node_face_connectivity:
+                    node_face_connectivity[j] = []
+                node_face_connectivity[j].append(face_idx)
+
+        # compare the two methods
+        for i in range(grid_ux.nMesh2_node):
+            face_index_from_sparse_matrix = grid_ux.Mesh2_node_faces.values[i]
+            valid_face_index_from_sparse_matrix = face_index_from_sparse_matrix[
+                face_index_from_sparse_matrix !=
+                grid_ux.Mesh2_node_faces.attrs["_FillValue"]]
+            valid_face_index_from_sparse_matrix.sort()
+            face_index_from_dict = node_face_connectivity[i]
+            face_index_from_dict.sort()
+            self.assertTrue(
+                np.array_equal(valid_face_index_from_sparse_matrix,
+                               face_index_from_dict))
```

Comment:
Changing the indentation from Line 843 to 865 (i.e. removing those from the for loop) seems to be damaging the fat that the for loop is running assertions for multiple grid paths in `grid_paths`.

Reply:
Good catch! Must of been an artifact from a merge conflict.